tests/test_impersonation.py

```diff
@@ -135,6 +136,8 @@ def test_impersonation_custom(self):
             'exec_python_fn'
         )
 
+    @unittest.skipIf(conf.get('core', 'executor') == 'SequentialExecutor',
```

Comment:
Add logic in `SubDagOperator` to raise exception when `SequentialExecutor` is used.

Reply:
Throw exception when backfilling `DAG`s that contains a `SubDagOperator` with a `SequentialExecutor`.